viewer/config.txt

```diff
@@ -19,26 +19,27 @@ Center Frame         : true
 Frame Maximized      : false
 Save Frame State     : true
 
-Overlay Default Visibility:
+# Overlay Default Visibility
 Server Speed         : true
 Foul Overlay         : true
 Field Overlay        : true
 Number of Players    : true
 Player IDs           : true
 
-Networking Settings:
+# Networking Settings
 Auto-Connect         : true
 Auto-Connect Delay   : 1000
-Server Hosts         : localhost
-Server Port          : 3200
+Default Server Host  : localhost
+Default Server Port  : 3200
```

Comment:
For consistency with the `Server` line it would probably be best if this was combined into a single `Default Server : localhost:3200`.